bool isEnabled

float64 motorPositionA
float64 motorPositionB